base/linalg/schur.jl

```diff
@@ -61,6 +61,15 @@ function getindex(F::Schur, d::Symbol)
     end
 end
 
+function show(io::IO, F::Schur)
+    println(io, "$(typeof(F)) with factors T and Z:")
+    show(io, F[:T])
+    println(io)
+    show(io, F[:Z])
+    println(io, "and values:")
```

Comment:
is this going to end up on the same line as the Z factor? possibly without any spacing between the factor array and the "and " ... ?